source/org/zfin/sequence/ForeignDB.java

```diff
@@ -196,22 +196,32 @@ public static AvailableName getType(String type) {
         }
     }
 
-    public boolean isZfishbook () {
+    public boolean isZfishbook() {
         if (displayName != null && displayName.equals("zfishbook"))

```

Comment:
This is a small tweak, but what do you think of using:

`"zfishbook".equals(displayName)`

instead of 

`displayName != null && displayName.equals("zfishbook")`